tox.ini

```diff
@@ -57,7 +57,7 @@ passenv =
 commands =
     coverage run --source sagemaker -m pytest {posargs}
     {env:IGNORE_COVERAGE:} coverage report --fail-under=90 --omit */tensorflow/tensorflow_serving/*
-deps = .[test]
+extras = test
```

Comment:
what caused this change?

Reply:
With the old configuration, the dependencies were not updated for me locally. It might be my setup. this way is more reliable.